src/repositories/CancelableResponse.ts

```diff
@@ -54,3 +54,9 @@ export class CancelableResponse<Data> {
 const noop = () => {
     return;
 };
+
+export type Canceler = (message?: string) => void;
+
+export function isCancel(error: any) {
+    return Axios.isCancel(error) || (error && error.name === "AbortError");
+}
```

Comment:
Export ``isCancel`` method

Reply:
Could go as `api.isCancel()` (which uses the repository), but that's also ok.